test/run-stp-tests.sh

```diff
@@ -776,6 +775,11 @@ echo "    $date by $user on `hostname`." >>$strfile
 echo "    <pre>" >>$strfile
 
 for file in 5*.sh; do
+	#
```

Comment:
Purpose of this change? Would be useful to know if it is necessary...

Reply:
This is bf16ca746ed902bc2c55165ea7f4bd2c1a112ec2 (This PR is a suite of 12 different commits);
> Tests: Make sure that all scheduled jobs are finished before running a new test. Fixes failures on slow architectures like armel and mipsel.
    Bug-Debian: https://bugs.debian.org/670878

(Past) experience has shown that on very slow architectures (such as these), waiting extensively at this point for all past jobs to be properly terminated, helped.